### PR TITLE
Export needs to use single Relativizer for all parts of the archive

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
Fixes exports of samples and protocol applications with material inputs which are saved by separated savers.

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

**Comment by labkey-matthewb** [comment]
Is it OK to always use LSIDRelativizer.FOLDER_RELATIVE?

**Review by labkey-jeckels — APPROVED** [review]
This looks good. Did you make changes to LSIDRelativizer or just move it? I'm assuming the latter but I can't easily tell in the diff.

**Comment by labkey-matthewb** [comment]
> This looks good. Did you make changes to LSIDRelativizer or just move it? I'm assuming the latter but I can't easily tell in the diff.

Just moved it to make it visible to FolderaExportContext.  And hard-coded some strings to avoid having to pull along even more files

**Comment by labkey-jony** [comment]
I tested this locally, but ran into the same issue that was reported, so I'm not sure what is going wrong here.

File: api/src/org/labkey/api/exp/XarSource.java [hunk 1]
```diff
@@ -202,7 +202,7 @@ private String createIllegalReferenceMessage(ExpRun experimentRun, ExpProtocolAp
         return message;
     }
 
-    public ExpMaterial getMaterial(ExpRun experimentRun, ExpProtocolApplication protApp, String materialLSID) throws XarFormatException
+    private ExpMaterial getMaterialOLD(ExpRun experimentRun, ExpProtocolApplication protApp, String materialLSID) throws XarFormatException
```
Comment (labkey-jeckels) [inline api/src/org/labkey/api/exp/XarSource.java]: Can we delete this now?

**Review by labkey-jeckels — APPROVED** [review]
Looks OK to me.

**Pull request merged**